### PR TITLE
Update to GF African Latin

### PR DESCRIPTION
**Pull request by LauraMeseguer**
Completed encodings

GF-latin-expert_unique-glyphs.nam
GF-latin-core_unique-glyphs.nam
GF-latin-plus_optional-glyphs.nam
GF-latin-plus_unique-glyphs.nam
GF-latin-pro_optional-glyphs.nam
GF-latin-pro_unique-glyphs.nam
GF-latin-AfricanLatinLanguages#177.nam

Extra added glyphs to the Pan African Latin (validate and say ok or not to include in the corresponding GF-latin-AfricanLatinLanguages#177.nam )

rfishhook
gravecomb_macroncomb
acutecomb_macroncomb
macroncomb_gravecomb
macroncomb_acutecomb

Total
1355 glyphs

What is left?

-Final review of the spacing and kerning groups
-Final go or not to create a new GF-latin-AfricanProLatinLanguage from the existing GF-latin-AfricanLatinLanguages#177.nam that will include 168 Small Caps of which 137 need to be built

### CODE REVIEW

**Comment by graphicore** [comment]
Thanks a lot @LauraMeseguer 
Note that I just reacted the proper way to @davelab6 request #2. Dave will go on to transfer this repo to the googlefonts organization. I think no further action on your side nor on this PR will be needed for that.

**Comment by davelab6** [comment]
@moyogo please could you review the african part of thisi?

**Comment by davelab6** [comment]
Transfer complete, new repo URL is https://github.com/googlefonts/questrial

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,5 @@
 # font editor temporary files
 *(Autosaved).glyphs
 *.vfbak
+fonts/
```
Comment (davelab6) [inline .gitignore:8]: Per https://github.com/googlefonts/gf-docs/tree/master/Spec we should have a `/fonts/` folder with the VF and `/fonts/ttf` folder with statics

**Comment by davelab6** [comment]
@LauraMeseguer please pull your master branch to get the commit I made that updates your `/.gitignore` file, and then add and commit the `/fonts` folder, then I'll merge :)

**Comment by LauraMeseguer** [comment]
@davelab6 Do you need the binary ttf and otf files committed to the repo or just an empty fonts folder?

**Comment by moyogo** [comment]
a vs alpha-latin, alpha-latin can have a hook-terminal like Alpha-latin to improve distinction with a. That said, users of languages that use both a and alpha-latin will probably avoid fonts with single storey-a.
image

florin is the lowercase of Fhook and should be upright like f in Roman style, with ascender-height top and descender-height bottom hook.
image

Gammaafrican and gamma-latin are fine. I would have made the loops a bit bigger to make them more round.
image

Jcrossedtail, jcrossedtail could have rounder loops, less wide
image

Nhookleft and Eng should either both n-form or both N-form. The n-form is used in most languages using both or those using Eng. The lowercase eng should have a hook descender instead of a straight descender. The straight descender matches the style of the j but it’s problematic for the eng.
image

thook is confusable with f, I would add a baseline hook
image

Ezh and ezh: the glyphs can be wider, closer to Z and z, and the center angle can be shifted down to middle a bit like:
image
image

Upsilonafrican and upsilon-latin are fine, but they most commonly have the horizontal bars directly connected to the bowl:
image
image

Glottalstop can have a wider and larger bowl to better match other uppercase letters. Right now it’s too close to the lowercase.
image

glottalstopmod can have a longer stem, sharing the same baseline as hmod, zmod, like a superscript glottalstop.
image
image

colonmod is better if its dots are closer together than those of colon
image

Khook, Whook, Yhook, whook, yhook need wider right side margin, or some kerning as they clash in some pairs.
image


Is there too much contrast on the cedillacomb, brevecomb, brevebelowcomb, breveinvertedcomb, breveinvertedbelowcomb, ogonekcomb, tildecomb, tildebelowcomb, tildeoverlaycomb for the this style?
image
image

**Comment by davelab6** [comment]
> users of languages that use both a and alpha-latin will probably avoid fonts with single storey-a.

Should we have 2 story 'a' that can be accessed by `locl` and `ss01`?

**Comment by davelab6** [comment]
> `/documentation/Questrial_GF-latin-core_unique-glyphs.pdf`

I suggest to add a `/documentation/README.md` file that explains briefly how they were made :)

**Comment by LauraMeseguer** [comment]
@moyogo Doesn't the glottalstopmod has to be aligned to the hmod on the top?

**Comment by LauraMeseguer** [comment]
> Could the italic f be assigned to U+1D453, so we don't lose the design (perfectly valid for U+1D453, even if it isn't for U+0192)?

Sure

**Comment by davelab6** [comment]
Thanks @thlinard for the tip! 

@LauraMeseguer I have merged this :) 

@moyogo and all, please file new issues for any further changes.

**Comment by moyogo** [comment]
I’ll open a new issue with a couple of notes and word sample for the kerning (including new ones I found).

**Comment by vv-monsalve** [comment]
I made a PR to Laura's master with the latest version of the files after some QA fixes. They would be almost ready for publishing.

It would be needed the Issue with notes from @moyogo  to include the last adjustments :)

Some of the adjustments done or needed are:

- To solve some fails, I deleted the note about "Reserved Font Name" from the Font Info (a description custom parameter as well as the copyright field. 
- Laura Meseguer credit was added in the Contributors file. Should she be included also as an author in the Designer field of the Font Info and the Authors.txt file?.
- The link http://www.admixdesigns.com/ in the Font info is broken. @davelab6 do you have contact with the original designer so we can restore it? I've looked up him on the internet but didn't find him, his latest post are from 2014.
- The GitHub URL for the project is missing on the Description file. Should we include Laura's repository, Google's one, or leave it like that? (meanwhile, I used her repo).
- Diffenator pics of the principal changes included [here](https://github.com/vv-monsalve/questrial/issues/3).
- Browser previews [here](https://github.com/vv-monsalve/questrial/issues/4).

**Comment by davelab6** [comment]
@moyogo please post the issue :)

**Pull request merged**